gekitchen/erd/registry.py

```diff
@@ -0,0 +1,85 @@
+from typing import TypedDict
+
+from .codes import ErdCode
+from .converters import *
+
+class ConverterRegistry(TypedDict):
+    key: ErdCode
+    value: ErdValueConverter
```

Comment:
If all of these methods end up being decorated `@staticmethod` or `@classmethod`, you can change this to `Type[ErdValueConverter]` and get rid of all of the instance creation below.  Alternatively, if you use a `Protocol` instead of an `ABC`, both instance and classes with static/class methods can be hinted as `ErdConverterProto` or whatever you call it.

Reply:
I think some of this might also be applicable in the new version, but I did restructure it even further and made things into configuration entries.  We should look to see if these changes would help there too.